alert_tools/tools.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from astropy.time import Time
+from astropy.io import fits
+import astropy.units as u
+import aplpy
```

Comment:
We're not using this library in this script so you can remove this import for now.